src/renderers/testing/ReactTestRenderer.js

```diff
@@ -36,13 +36,21 @@ function getRenderedHostOrTextFromComponent(component) {
 }
 
 
+var defaultMockConfig = {
+  getMockRef: function() {
+    return {};
+  },
+}
+
 // =============================================================================
 
 var ReactTestComponent = function(element) {
   this._currentElement = element;
   this._renderedChildren = null;
   this._topLevelWrapper = null;
+  this._mockConfig = defaultMockConfig;
```

Comment:
Can you please make it always read the mock config from transaction? By providing a default, I meant doing it at top level. So if you don’t pass `{ getMockRef }` to `ReactTestRenderer.create`, you get a default implementation *at that point*.